hack/k8s-client-go.patch

```diff
@@ -0,0 +1,12 @@
+--- a/vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go
++++ b/vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go
+@@ -290,7 +290,8 @@ func (le *LeaderElector) release() bool {
+ 		return true
+ 	}
+ 	leaderElectionRecord := rl.LeaderElectionRecord{
+-		LeaderTransitions: le.observedRecord.LeaderTransitions,
++		LeaderTransitions:    le.observedRecord.LeaderTransitions,
++		LeaseDurationSeconds: le.observedRecord.LeaseDurationSeconds,
+ 	}
+ 	if err := le.config.Lock.Update(context.TODO(), leaderElectionRecord); err != nil {
+ 		klog.Errorf("Failed to release lock: %v", err)
```

Comment:
I see we do this in eventing, I'd keep track of it in an issue so we remember to get rid of it once the upstream issue is fixed, maybe linking the [PR](https://github.com/kubernetes/kubernetes/pull/91942), WDYT?